build/docker/copy-server-config.sh

```diff
@@ -23,16 +23,13 @@ cp -pr ${WORKSPACE}/fhir-server/liberty-config/config/default/fhir-server-config
 
 echo "Replacing datasource content in server configDropins..."
 OVERRIDES="${WORKSPACE}/build/docker/fhir-server/volumes/overrides"
-mkdir -p $OVERRIDES
+mkdir -p ${OVERRIDES}
 
 # Just in case it already exists, let's wipe the datsource*.xml files
-rm -f $OVERRIDES/datasource*.xml 2> /dev/null
+rm -f ${OVERRIDES}/datasource*.xml 2> /dev/null
```

Comment:
The stderr redirect is redundant here. Ought to reference rm by its full path, just in case rm gets aliased.

Reply:
OK.  I just added the curly braces to make it consistent with other var references (would also be fine removing them from each one since they aren't needed)...was just going for consistency.

> Ought to reference rm by its full path, just in case rm gets aliased.
that seems a bit overkill to me...and I don't think we do that anywhere else.

```suggestion
rm -f ${OVERRIDES}/datasource*.xml
```